docs/gsoc.rst

```diff
@@ -184,6 +184,15 @@ which provide outlines on the larger tasks that we're hoping to accomplish.
 .. _these issues: https://github.com/readthedocs/readthedocs.org/issues?direction=desc&labels=Feature&page=1&sort=updated&state=open
 .. _milestones: https://github.com/readthedocs/readthedocs.org/milestones
 
+Some previous year project selected in Google Summer Of Code  
```

Comment:
I think the heading can be improved to something like: `Previous Years Projects`.
Currently, `Some` suggests that some projects are not listed.